app/app.go

```diff
@@ -391,6 +392,16 @@ func (app *App) Update(updateData App, w io.Writer) (err error) {
 	if tags != nil {
 		app.Tags = tags
 	}
+	if platform != "" {
+		p, errPlat := GetPlatform(platform)
+		if errPlat != nil {
+			return errPlat
+		}
+		app.Platform = p.Name
```

Comment:
I think we could check here if app.Platform is different then p.Name and only if thats true, set app.UpdatePlatform to true.